src/options.c

```diff
@@ -398,6 +400,7 @@ options_init(options_t *options)
 	options->provider = NULL;
 
 	options->use_fade = -1;
+    options->fade_duration = -1;
```

Comment:
indent

Reply:
Thanks, fixed. I also adjusted CLion IDE to better detect existing indents in files.